packages/ui/src/components/Form/expression/ExpressionEditor.tsx

```diff
@@ -79,8 +79,11 @@ export const ExpressionEditor: FunctionComponent<ExpressionEditorProps> = ({
 
     // Filter menu items based on the text input value when one exists
     if (filterValue) {
-      newSelectOptions = languageCatalogMap.filter((menuItem) =>
-        String(menuItem.value).toLowerCase().includes(filterValue.toLowerCase()),
+      newSelectOptions = languageCatalogMap.filter(
+        (menuItem) =>
```

Comment:
Just an extremely 💅 💅 , maybe we could simply lowercase the `filterValue` once before using it in each filter pass, we can do it in another PR 👌